webpack.config.js

```diff
@@ -29,6 +29,7 @@ const entry = devMode
       examples: path.resolve(__dirname, "./src/index.js"),
     }
   : {
+     "book-byen": path.resolve(__dirname, "./src/book-byen/book-byen.js"),
      twentyThreeVideo: path.resolve(__dirname, "./src/twenty-three-video/twenty-three-video.js"),
```

Comment:
Please keep entries consistent. It seems like kebab-case is the default. In that case it should be something like `twenty-three-video`.